pnpm-lock.yaml

```diff
@@ -8128,7 +8128,7 @@ snapshots:
 
   '@electron/get@2.0.2':
     dependencies:
-      debug: 4.3.7
+      debug: 4.4.0
```

Comment:
`debug` and `cookie` got updated for some reason anyway. 😭

Reply:
Yeah... If I just go and update the electron version on master, they get updated too. Seems like an issue in pnpm.
But I think I will keep it as is, it doesn't hurt us to have them updated.

Reply:
Sounds good. 👌

Reply:
I brought up something similar in https://github.com/pnpm/pnpm/issues/5552#issuecomment-2428653185, you can upvote it I guess. ;)